src/scaffold/template/base/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "{{ projectName }}",
+  "name": "{{ npmPackage }}",
```

Comment:
This is mapped, for example to: @org.acme.outside/garden-pet-stream-api. Is this really the right name structure to use here? I have no idea if there exists convention about this, I only found [this](https://docs.npmjs.com/creating-a-package-json-file). 

> The "name" field contains your package’s name, and must be lowercase and one word, and may contain hyphens and underscores.

Reply:
Maybe not... I will let this be reviewed also at architecture level.
Scopes are described in https://docs.npmjs.com/about-scopes

Reply:
After discussions at architecture level, we replace dots with hyphens. This is compliant with the rules you mention and practice we can observe on npmjs.org.